README.md

```diff
@@ -105,6 +105,7 @@ __fastify-gql__ supports the following options:
   exposed at `/graphql`.
 * `prefix`: String. Change the route prefix of the graphql endpoint if enabled.
 * `defineMutation`: Boolean. Add the empty Mutation definition if schema is not defined (Default: `false`).
+* `errorHandler`: Function. Change the default error handler.
```

Comment:
I think this  should also be `false`, meaning that no errorHandler would be installed.

Reply:
And maybe add details about the standardized response that the user should implement if a customized handler is defined ?

Reply:
Yes please.